src/handlers/user-identity/tests/user-identity-handler.test.ts

```diff
@@ -157,3 +163,114 @@ describe("user-identity-handler tests", () => {
     });
   });
 });
+
+describe("user-identity-handler expired", () => {
+  const TEST_FRAUD_VALIDITY_HOURS: number = 4320; // ~6 months
+  const NOW: string = "2025-08-24T15:35:58.000Z";
+  const NOT_EXPIRED_NBF: string = "2025-05-15T16:30:04.000Z";
+  const EXPIRED_NBF: string = "2025-01-12T10:02:54.000Z";
+  const RANDOM_NBF: string = "2023-04-25T15:01:36.000Z";
+
+  const FRAUD_ISSUER = "fraudCRI";
+  const PASSPORT_ISSUER = "passportCRI";
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    newEvent = event();
+    jest.spyOn(configuration, "getServiceApiKey").mockResolvedValue("an-api-key");
+    jest
+      .spyOn(configuration, "getConfiguration")
+      .mockResolvedValue({ evcsApiUrl: "https://evcs.gov.uk" } as Configuration);
+    jest.spyOn(configuration, "getConfiguration").mockResolvedValue({
+      fraudIssuer: [FRAUD_ISSUER],
+      fraudValidityPeriod: TEST_FRAUD_VALIDITY_HOURS,
+    } as Configuration);
+    jest.spyOn(fraudCheckService, "hasFraudCheckExpired").mockRestore();
+
+    jest.useFakeTimers();
+    jest.setSystemTime(new Date(NOW));
+  });
+
+  it.each([
+    { fraudCheckInputs: [{ nbf: NOT_EXPIRED_NBF, state: State.CURRENT }], expectedExpired: false },
+    { fraudCheckInputs: [{ nbf: EXPIRED_NBF, state: State.CURRENT }], expectedExpired: true },
+    {
+      fraudCheckInputs: [
+        { nbf: NOT_EXPIRED_NBF, state: State.CURRENT },
+        { nbf: EXPIRED_NBF, state: State.HISTORIC },
+      ],
+      expectedExpired: false,
+    },
+    {
+      fraudCheckInputs: [
+        { nbf: EXPIRED_NBF, state: State.CURRENT },
+        { nbf: NOT_EXPIRED_NBF, state: State.HISTORIC },
+      ],
+      expectedExpired: true,
+    },
+  ])(`should set expired value based on nbf of CURRENT fraud check`, async ({ fraudCheckInputs, expectedExpired }) => {
```

Comment:
As NBF and an acronym:
```suggestion
  ])(`should set expired value based on NBF of CURRENT fraud check`, async ({ fraudCheckInputs, expectedExpired }) => {
```